<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vote</class>
 <widget class="QDialog" name="vote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VOTE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="pmln_rb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PML-N</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pti_rb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PTI</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pml_rb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PML</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ppp_rb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PPP</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mqm_rb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MQM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnic_input">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>143</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CNIC :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submitbtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>☑️ Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
